sway/commands/gaps.c

```diff
@@ -0,0 +1,153 @@
+#include <string.h>
+#include "sway/commands.h"
+#include "sway/config.h"
+#include "sway/tree/arrange.h"
+#include "log.h"
+#include "stringop.h"
+
+int read_double(char *str, double *out) {
```

Comment:
This prototype should read as `static int read_double(const char *str, double *out)`

Reply:
I'd also prefer that `bool` is used instead of `int`. It would actually be possible to just use `static double read_double(const char *str)` and use NaN as an invalid return, but it's not something that's important.